modules/nf-core/yak/count/meta.yml

```diff
@@ -0,0 +1,45 @@
+name: "yak_count"
+description: build k-mer hash table for assembly; count singletons
+keywords:
+  - k-mers
+  - count
+  - assembly
+
+tools:
+  - "yak":
+      description: "Yet another k-mer analyzer"
+      homepage: "https://github.com/lh3/yak"
+      documentation: "https://github.com/lh3/yak/blob/master/README.md"
+      tool_dev_url: "https://github.com/lh3/yak"
+      licence: ["MIT"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - fastqs:
+      type: path
+      description: directory contains compressed reads fastq file
+      pattern: "*.{fastq.gz,fq.gz}"
```

Comment:
Can also be fasta.

Reply:
thanks, I have added fasta option as you advised.